components/script/dom/performance.rs

```diff
@@ -225,19 +240,19 @@ impl Performance {
         // Step 4.
         // If the "add to performance entry buffer flag" is set, add the
         // new entry to the buffer.
+
         if add_to_performance_entries_buffer {
             self.entries
                 .borrow_mut()
                 .entries
                 .push(DomRoot::from_ref(entry));
+            return;
```

Comment:
We shouldn't bail out here. This wasn't part of the original patch from #22431.

Reply:
Should I update the test-expectations log  after this change?

Reply:
Yes, it may be needed.